src/entity-manager/EntityManager.ts

```diff
@@ -356,13 +356,45 @@ export class EntityManager {
                 .whereInIds(criteria)
                 .execute();
 
-        } else {
-            return this.createQueryBuilder()
-                .update(target)
-                .set(partialEntity)
-                .where(criteria)
-                .execute();
-        }
+    } else {
+     return this.createQueryBuilder()
+            .update(target)
+            .set(partialEntity)
+            .where(criteria)
+            .execute();
+    }
+
+
+    }
+
+    /**
+     * Restores a given entity.
```

Comment:
Provide a better comment, mention that it restores entities marked with SoftDelete column

Reply:
Also what happens user specifies entity without soft delete column here?

Reply:
Checking